src/app/Notifications/Notifications.tsx

```diff
@@ -117,6 +146,20 @@ export class Notifications {
     }
     this._notifications$.next(this._notifications);
   }
+
+  private isActionNotification(n: Notification): boolean {
+    return (n.category !== 'WsClientActivity' && n.category !== 'TargetJvmDiscovery')
+      && (n.variant === AlertVariant.success || n.variant === AlertVariant.info);
```

Comment:
This ends up being basically any notification than isn't a networkInfo or a problem, right? Could it be re-written in those terms?

Reply:
The following edit prevents the notification drawer from displaying any notifications, including errors. Will keep looking into it.
```
private isActionNotification(n: Notification): boolean {
    return !this.isNetworkInfoNotification(n) && !this.isProblemNotification(n);
}
```

Reply:
`a.filter(this.isActionNotification)` was causing an `Uncaught TypeError: Cannot read properties of undefined (reading 'isNetworkInfoNotification')`. Replacing `a.filter(this.isActionNotification)`  with `a.filter(n => this.isActionNotification(n))` resolves the type error.